config/crd/bases/infrastructure.cluster.x-k8s.io_baremetalmachines.yaml

```diff
@@ -204,6 +204,10 @@ spec:
       description: BaremetalMachine is Ready
       name: Ready
       type: string
+    - JSONPath: .metadata.labels.cluster\.x-k8s\.io/cluster-name
+      description: Cluster to which this BMCluster belongs
```

Comment:
same here, re-generate with the fixed types to fix it.

Reply:
fixed